service/frontend/adminHandler.go

```diff
@@ -247,7 +247,7 @@ func (adh *adminHandlerImpl) DescribeWorkflowExecution(
 		return nil, adh.error(err, scope)
 	}
 
-	shardID := common.WorkflowIDToHistoryShard(*request.Execution.WorkflowID, adh.numberOfHistoryShards)
+	shardID := common.WorkflowIDToHistoryShard(request.Execution.WorkflowID, adh.numberOfHistoryShards)
 	shardIDstr := string(shardID)
```

Comment:
I noticed on the domain-id diff: https://github.com/uber/cadence/blob/master/service/frontend/adminHandler.go#L940-L948
lower in this file, you can drop the `Get...` calls too

Reply:
I agree with all you comments, but as you can see, this PR is already pretty big. I wanted to avoid additional changes like these to keep it mostly mechanical, even though such simplifications are tempting. 

Once this is merged we can do another pass to drop `Get...` helpers as well, further simplifying our code.

Reply:
yep, totally agreed 👍